src/libs/migrations/PersonalDetailsByAccountID.js

```diff
@@ -0,0 +1,213 @@
+                        Log.info(`[Migrate Onyx] PersonalDetailsByAccountID migration: removing reportAction ${reportActionID} because accountID not found`);
+                        return;
+                    }
+                }
+
+                if (lodashHas(reportAction, ['actorEmail']) && !lodashHas(reportAction, ['actorAccountID'])) {
+                    reportActionsWereModified = true;
+                    const actorAccountID = _.get(oldPersonalDetails, [reportAction.actorEmail, 'accountID']);
+                    if (actorAccountID) {
+                        newReportAction.actorAccountID = actorAccountID;
+                    } else {
+                        Log.info(`[Migrate Onyx] PersonalDetailsByAccountID migration: removing reportAction ${reportActionID} because actorAccountID not found`);
+                        return;
+                    }
+                }
+
+                if (lodashHas(reportAction, ['childManagerEmail']) && !lodashHas(reportAction, ['childManagerAccountID'])) {
+                    reportActionsWereModified = true;
+                    const childManagerAccountID = _.get(oldPersonalDetails, [reportAction.childManagerEmail, 'accountID']);
+                    if (childManagerAccountID) {
+                        newReportAction.childManagerAccountID = childManagerAccountID;
+                    } else {
+                        Log.info(`[Migrate Onyx] PersonalDetailsByAccountID migration: removing reportAction ${reportActionID} because childManagerAccountID not found`);
+                        return;
+                    }
+                }
+
+                if (lodashHas(reportAction, ['whisperedTo']) && !lodashHas(reportAction, ['whisperedToAccountIDs'])) {
+                    reportActionsWereModified = true;
+                    const whisperedToAccountIDs = [];
+                    _.each(reportAction.whisperedTo, (whisperedToLogin) => {
+                        const whisperedToAccountID = _.get(oldPersonalDetails, [whisperedToLogin, 'accountID']);
+                        if (whisperedToAccountID) {
+                            whisperedToAccountIDs.push(whisperedToAccountID);
+                        }
+                    });
+
+                    if (whisperedToAccountIDs.length === reportAction.whisperedTo.length) {
+                        newReportAction.whisperedToAccountIDs = whisperedToAccountIDs;
+                    } else {
+                        Log.info(`[Migrate Onyx] PersonalDetailsByAccountID migration: removing reportAction ${reportActionID} because whisperedToAccountIDs not found`);
+                        return;
+                    }
+                }
+
+                if (lodashHas(reportAction, ['childOldestFourEmails']) && !lodashHas(reportAction, ['childOldestFourAccountIDs'])) {
+                    reportActionsWereModified = true;
+                    const childOldestFourEmails = reportAction.childOldestFourEmails.split(',');
```

Comment:
NAB, it's strange that this is a string instead of an array. Just noting in case you want to add this to a todo list somewhere.